main.go

```diff
@@ -226,10 +226,6 @@ func (o *Options) Validate() error {
 		return fmt.Errorf("must specify --database-dsn with --load-database")
```

Comment:
Won't database-dsn always be required now? Maybe we should error if not specified?

Reply:
I don't think we need it for fetch data, but I can expand this to server and loaddatabase. Seem right?